source/text/labelgeometry.ts

```diff
@@ -44,26 +39,13 @@ export class LabelGeometry extends Geometry {
      * |    \  |
      * 1-------3
      */
-    protected _vertices: Float32Array = new Float32Array(0);
-    /**
-     * Texture coordinates (uv) for every glyph, format: ll.x, ll.y, ur.x, ur.y
-     */
-    protected _texCoords: Float32Array = new Float32Array(0);
-    /**
-     * The coordinates of the lower left corner of every glyph. Its interpretation depends on the shader,
-     * usually it's a 3-component vector in world space.
-     */
-    protected _origins: Float32Array = new Float32Array(0);
-    /**
-     * The tangent vector for every glyph (direction along base line). Its interpretation depends on the shader,
-     * usually it's a 3-component vector in world space.
-     */
-    protected _tangents: Float32Array = new Float32Array(0);
+    protected static readonly DATA = new Float32Array([0, 0, 0, 1, 1, 0, 1, 1]);
```

Comment:
Is it common knowlegde that vertices are called DATA? I find DATA quite generic. I don't recommend to use `VERTICES` instead, because that could mislead to think that the glyphs' origins are stored here. But maybe something like QuadVertices, QuadData, QuadCorners...  it's not that important to me, it is just a thought,

Reply:
ah - i copied the name from another project ... :smile: - the default is `VERTICES` - thanks